src/Chunk.cpp

```diff
@@ -1715,10 +1715,12 @@ void cChunk::SetAlwaysTicked(bool a_AlwaysTicked)
 	if (a_AlwaysTicked)
 	{
 		m_AlwaysTicked += 1;
+		Stay(a_AlwaysTicked);
```

Comment:
In other words, the call to `Stay` may either be made independently, or implicitly through a call to `SetAlwaysTicked`. It's simply weird to want an always ticking but not always loaded chunk. The other way around is still possible.